app/serializers/success_serializer.rb

```diff
@@ -0,0 +1,16 @@
+class SuccessSerializer
```

Comment:
Secondo me vanno restituiti sempre i model/collection dell'oggetto in considerazione, mai un messaggio generico di ok